processing/src/main/java/io/druid/query/groupby/GroupByQueryQueryToolChest.java

```diff
@@ -414,6 +415,8 @@ public String apply(DimensionSpec input)
     return new CacheStrategy<Row, Object, GroupByQuery>()
     {
       private final List<AggregatorFactory> aggs = query.getAggregatorSpecs();
+      private final List<DimensionSpec> dims = query.getDimensions();
+
 
       @Override
       public byte[] computeCacheKey(GroupByQuery query)
```

Comment:
either we need to change the cache key or tell people to flush their cache to avoid having errors with the old format when upgrading.